docs/user-guide/src/version_support.md

```diff
@@ -0,0 +1,46 @@
+# Supported release versions
+
+The Cluster API Provider Metal3 team maintains the following branches for CAPM3 for different API versions.
```

Comment:
```suggestion
The Cluster API Provider Metal³ team maintains the following branches for CAPM3 for different API versions.
```

Reply:
Actually we dont right it that way in the repo itself @furkatgofurov7

Reply:
I was referring to https://github.com/metal3-io/metal3-docs#what-is-metal

Reply:
I am referring this https://github.com/metal3-io/cluster-api-provider-metal3#what-is-the-cluster-api-provider-metal3

Reply:
yes, so it is a matter of choice in that case

Reply:
Yeah lets keep it as it is